### ISSUE TITLE
db_QueryDelete, db_QueryInsert should be renamed

### ISSUE THREAD

**Issue by povrazor** [ludumdare/ludumdare#1234]
`db_QueryDelete` and `db_QueryInsert` don't do exactly what they say. They attempted to be clever, to return values useful to the mentioned query types, but they are essentially the same thing as `db_Query`.

`db_QueryDelete` returns the number of affected fields.
`db_QueryInsert` returns the id of the item inserted (or first item changed).

I recently added some SQL code that pushed **A LOT** of data in to a table using an `INSERT` query, which is a completely valid use case, but functionally different than `db_QueryInsert` is used for. Ultimately I ended up using `db_QueryDelete`.

`db_QueryDelete` should probably be named `db_QueryChanged`, and `db_QueryInsert` become `db_QueryId`. The rest of the `db_Query` family should be evaluated, and make sure that the word following "Query" describes what is being returned. The name `db_QueryFetch` makes sense.

**Comment by local-minimum** [comment]
`db_QueryAffectedFields` `db_QueryAffectedItemId` ?

**Comment by sgstair** [comment]
Yeah, I ran into this with multiple-insert in notifications too. Will want to revisit the insert query there too. There isn't a nice structured way to do multiple insert at the moment without just generating the query manually.
